### PR TITLE
Setup RabbitMQ with docker compose

### PR DESCRIPTION
**Pull request by vadymtomashchuk18**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,4 +58,24 @@ docker exec -it postgres13 psql --username=cmltemplate --dbname=cmltemplate
...
 
-Run either with `--cors.enabled=true` program argument OR with `CORS_ENABLED=true` environment variable. 
\ No newline at end of file
+Run either with `--cors.enabled=true` program argument OR with `CORS_ENABLED=true` environment variable. 
+
+### Run RabbitMQ Cluster
+
+Before run cluster the network shared by all Rabbit containers should be created by this command:
+```                                        
+docker network create rabbitmq-cluster
```
Comment (xonixx) [inline README.md:67]: Is this line required? It's not quite clear why we need to setup network for multiple containers if we only run one container.

File: src/main/resources/application.yml [hunk 2]
```diff
@@ -3,6 +3,8 @@ server:
   port: 8080
 
 spring:
+  main:
+    allow-bean-definition-overriding: true
```
Comment (xonixx) [inline src/main/resources/application.yml:7]: Why do we need this setting for configuting RabbitMQ?

File: README.md [hunk 3]
```diff
@@ -58,4 +58,24 @@ docker exec -it postgres13 psql --username=cmltemplate --dbname=cmltemplate
...
+Run RabbitMQ cluster:
+```
+docker-compose -f docker-compose-rabbitmq.yml up
+```
+
+To test how messaging works make this call in java code with appropriate data
+```
+rabbitTemplate.convertAndSend(routingKey, "Prepared message"); 
+```
+And listener will return to you this message: `Message read from the queue : Prepared message`
```
Comment (xonixx) [inline README.md:81]: Does the docker image include some admin UI for RabbitMQ? If yes - could we please document how to access it?

File: README.md [hunk 4]
```diff
@@ -58,4 +58,24 @@ docker exec -it postgres13 psql --username=cmltemplate --dbname=cmltemplate
...
+Before run cluster the network shared by all Rabbit containers should be created by this command:
+```                                        
+docker network create rabbitmq-cluster
+``` 
+
+`rabbitmq-cluster` network is specified in `docker-compose-rabbitmq.yml` file.
+
+Run RabbitMQ cluster:
+```
+docker-compose -f docker-compose-rabbitmq.yml up
```
Comment (xonixx) [inline README.md:74]: Could you please also add a shortcut to this command to `Makesurefile` the same as it's done for mysql/postgre?

The idea here to enable running this just with `./makesure rabbitmq`

**Review by xonixx — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (vadymtomashchuk18) [reply README.md:66]: Done

**Pull request merged**